holidays/countries/finland.py

```diff
@@ -97,6 +101,71 @@ def _populate_public_holidays(self):
         # Second Day of Christmas.
         self._add_christmas_day_two(tr("Tapaninpäivä"))
 
+    def _populate_unofficial_holidays(self):
+        if self._year >= 2012:
+            # Day of Finnish architecture and design, Birthday of Alvar Aalto.
+            self._add_holiday_feb_3(tr("Arkkitehtuurin ja muotoilun päiviä"))
+
+        if self._year >= 1854:
```

Comment:
I doubt that this holiday was created during Runeberg's life. 👀

Reply:
It was!

> Since Runeberg’s fiftieth birthday in 1854, he has been celebrated publicly on his birthday each year on 5 February. Runeberg celebrations gradually came to be held throughout the country, and by the mid 1860s the celebrations had taken on an established format, involving patriotic speeches, music and singing in unison.

https://wiki.aineetonkulttuuriperinto.fi/wiki/Runeberg_celebrations

> Runeberg Day was celebrated for the first time in 1854 in Runeberg’s hometown of Jakobstad when the poet turned 50.

https://www.facebook.com/valtioneuvosto/posts/pfbid04kcsDPUzyZMcF749hyj4Zi5qNqxpBaXyGdVBPvPjQpk3AjLb2j1Mct1CbkUCoN7zl

> From his 50th birthday onwards, local students in Porvoo began to celebrate his birthday on February 5th. They came to serenade him on his birthdays. When he fell ill in 1863, his health made headlines in newspapers across the country and also abroad. 
> ...
> As we mentioned, his birthday began to be publicly celebrated already during his lifetime. By the 1860s, these celebrations became quite formalized. There were formal speeches, plays, and music. 

https://www.finnwards.com/living-in-finland/runebergs-day/